filament/backend/src/opengl/platforms/PlatformCocoaGL.mm

```diff
@@ -256,6 +269,9 @@
 
 void PlatformCocoaGL::commit(Platform::SwapChain* swapChain) noexcept {
     [pImpl->mGLContext flushBuffer];
+
+    // This needs to be done periodically.
+    CVOpenGLTextureCacheFlush(pImpl->mTextureCache, 0);
```

Comment:
again I don't want to do this if there is no need. It should only be done if there are external images active.
also can you comment what this does and why this is needed periodically?

Reply:
This code is coming from [here](https://github.com/google/filament/blob/d00223332ea71322bb2eb6a0e79100da79bd83ff/filament/backend/src/opengl/platforms/PlatformCocoaTouchGL.mm#L172). Removing this flush is fine on my case. Please let me know do you want to remove this then I will remove it.